controllers/usersignup/usersignup_controller.go

```diff
@@ -736,7 +736,7 @@ func (r *Reconciler) sendDeactivatingNotification(logger logr.Logger, config too
 		}
 
 		notification, err := notify.NewNotificationBuilder(r.Client, userSignup.Namespace).
-			WithTemplate(notificationtemplates.UserDeactivating.Name).
+			WithTemplate(notificationtemplates.SandboxUserDeactivating.Name).
```

Comment:
Same here. Why hardcoded to Sandbox?

Reply:
replaced with constant [3df1eab](https://github.com/codeready-toolchain/host-operator/pull/754/commits/3df1eab38ec6e59824567e8d6c9d39bd6c43a3ff)
Again, this is just template name, actual fetching based on notification env is done while sending.